packages/@sanity/base/src/conditional-property/__tests__/useResolveConditionalProperty.test.tsx

```diff
@@ -0,0 +1,144 @@
+// eslint-disable-next-line import/no-unassigned-import
```

Comment:
@vicbergquist I'm not sure why we have this file here since looking at gitlense it was added by you a couple of weeks back. Should we remove this from this pr? 😄

Reply:
I see that it was added on the c7228d674a05df896cb0b6cfac7194660b6efb22 commit. I had just assumed that it would have been in your pr!

Reply:
The file name was changed, probably why it's included here. Maybe your rebase didn't go as it should have?